app/models/ticket.rb

```diff
@@ -75,6 +75,17 @@ def tickets_turnover
     tickets_sold * price
   end
 
+  def tickets_turnover_total(id)
+    tickets = TicketPurchase.where(ticket_id: id)
+    amount = 0
+    begin
```

Comment:
why do you need a transaction? :thinking:

Reply:
from ticket purchase model, i wanted the quantity and amout_paid values so i used it.